static/package.json

```diff
@@ -23,6 +23,7 @@
     "comment-json": "^4.1.1",
     "css-loader": "^3.4.2",
     "currency-symbol-map": "^5.0.1",
+    "dotenv": "^16.4.1",
```

Comment:
we can remove this package now, right?

Reply:
you'll need to re-run `npm i` in static to update the package-lock to fully remove this package